gen/plugins/com.salesforce/com.salesforce.plugin.smartsync.js

```diff
@@ -51,11 +59,19 @@ var getSyncStatus = function(syncId, successCB, errorCB) {
         );        
 };
 
+var MERGE_MODE = {
```

Comment:
smartsync.js is not a requirement for using this plugin, so defining the enum here